File: Other/OddEvenString.py
def oddEvenString(string):
	odd = ""
	even = ""
	for i in range(len(string)):
		if i%2 == 0:
			even += string[i]
		else:
			odd += string[i]
	print(even + " " + odd)

oddEvenString("Hello")
oddEvenString("Darkness")
oddEvenString("My")
oddEvenString("Old")
oddEvenString("Friend")

File: Other/OddEvenString.js
function oddEvenString(str){
	var odd = "";
	var even = "";
	for(var i=0; i<str.length; i++){
		if(i%2==0){
			even += str[i];
		} else{
			odd += str[i];
		}
	}
	console.log(even + " " + odd);
}

oddEvenString("Hello");
oddEvenString("World");